packages/communication-react/CHANGELOG.beta.md

```diff
@@ -4,77 +4,47 @@ This log was last generated on Tue, 23 May 2023 21:43:12 GMT and should not be m
-  - Filter the errors based on whether the side pane is currently showing the video effects side pane ([PR #3020](https://github.com/azure/communication-ui-library/pull/3020) by 97124699+prabhjot-msft@users.noreply.github.com)
-  - [AzureCommunicationCallWithChatAdapter] Creating call with chat adapter from state now take an options bag ([PR #2998](https://github.com/azure/communication-ui-library/pull/2998) by joshlai@microsoft.com)
-  - updating default extensions and commands when creating a new codespace container ([PR #2986](https://github.com/azure/communication-ui-library/pull/2986) by 79329532+alkwa-msft@users.noreply.github.com)
-  - Custom Video Effect tests ([PR #3059](https://github.com/azure/communication-ui-library/pull/3059) by 97124699+prabhjot-msft@users.noreply.github.com)
-  - Video effect errors test ([PR #3066](https://github.com/azure/communication-ui-library/pull/3066) by 97124699+prabhjot-msft@users.noreply.github.com)
-  - Show Video Effects errors in side pane ([PR #2995](https://github.com/azure/communication-ui-library/pull/2995) by 97124699+prabhjot-msft@users.noreply.github.com)
-  - Fix for tooltip getting hidden under Effects title in video effects pane ([PR #2968](https://github.com/azure/communication-ui-library/pull/2968) by 97124699+prabhjot-msft@users.noreply.github.com)
-  - [ChatMessageContent] Updating the warning text to be string | undefined ([PR #2998](https://github.com/azure/communication-ui-library/pull/2998) by joshlai@microsoft.com)
-  - Fix positioning of video effects picker tiles when there is a row with a differing count ([PR #3076](https://github.com/azure/communication-ui-library/pull/3076) by 2684369+JamesBurnside@users.noreply.github.com)
-  - Introduce mechanism to control local video tile styling for different formfactors. ([PR #3006](https://github.com/azure/communication-ui-library/pull/3006) by 94866715+dmceachernmsft@users.noreply.github.com)
-  - [MessageThread] Remove extra arguments on CancelEditCallback ([PR #2998](https://github.com/azure/communication-ui-library/pull/2998) by joshlai@microsoft.com)
+  - Creating call with chat adapter from state now take an options bag ([PR #2998](https://github.com/azure/communication-ui-library/pull/2998) by joshlai@microsoft.com)
   - Stabilize the scrolling behavior ([PR #2959](https://github.com/azure/communication-ui-library/pull/2959) by jiangnanhello@live.com)
-  - Fix breaking changes of create adapter from client ([PR #2998](https://github.com/azure/communication-ui-library/pull/2998) by jiangnanhello@live.com)
 
-## [1.5.1-beta.4](https://github.com/azure/communication-ui-library/tree/@azure/communication-react_v1.5.1-beta.4)
+
+## [1.5.1-beta.4](https://github.com/azure/communication-ui-library/tree/1.5.1-beta.4)
 
 Fri, 28 Apr 2023 23:16:11 GMT 
 [Compare changes](https://github.com/azure/communication-ui-library/compare/@azure/communication-react_v1.5.1-beta.3...@azure/communication-react_v1.5.1-beta.4)
```

Comment:
Nit: Could you fix the formatting of this link from the previous beta release as well

Reply:
done